samples/apps/spring-petclinic/patch/application-postgresql.properties.patch

```diff
@@ -0,0 +1,4 @@
+diff --git a/src/main/resources/application-postgres.properties b/src/main/resources/application-postgresql.properties
+similarity index 100%
+rename from src/main/resources/application-postgres.properties
+rename to src/main/resources/application-postgresql.properties
```

Comment:
is renaming really needed?

Reply:
yes, the spring-cloud-binding requires the type to be `postgresql` and currently it is just `postgres`

Reply:
well it didn't work before, but it was some number of attempts ago... let me check at the current state

Reply:
Ok, no need for that, after all... reverting

Reply:
Thx @pedjak!